parse/table_definitions_arbitrum/perennial/Market_v2_1_event_AccountPositionProcessed.json

```diff
@@ -0,0 +1,131 @@
+                        },
+                        {
+                            "internalType": "UFixed6",
+                            "name": "rewardAmount",
+                            "type": "uint256"
+                        },
+                        {
+                            "internalType": "UFixed6",
+                            "name": "positionFee",
+                            "type": "int256"
+                        },
+                        {
+                            "internalType": "UFixed6",
+                            "name": "keeper",
+                            "type": "uint256"
+                        }
+                    ],
+                    "indexed": false,
+                    "internalType": "struct LocalAccumulationResult",
+                    "name": "accumulationResult",
+                    "type": "tuple"
+                }
+            ],
+            "name": "AccountPositionProcessed",
+            "type": "event"
+        },
+        "contract_address": "SELECT market FROM ref('MarketFactory_v2_event_MarketCreated') GROUP BY market",
+        "field_mapping": {},
+        "type": "log"
+    },
+    "table": {
+        "dataset_name": "perennial",
+        "schema": [
+            {
+                "description": "",
+                "name": "account",
+                "type": "STRING"
+            },
+            {
+                "description": "",
+                "name": "fromOracleVersion",
+                "type": "STRING"
+            },
+            {
+                "description": "",
+                "name": "toOracleVersion",
+                "type": "STRING"
+            },
+            {
+                "description": "",
+                "name": "fromPosition",
+                "type": "STRING"
+            },
+            {
+                "description": "",
+                "name": "toPosition",
+                "type": "STRING"
+            },
+            {
+                "description": "",
+                "fields": [
+                    {
+                        "description": "",
+                        "name": "collateralAmount",
+                        "type": "STRING"
+                    },
+                    {
+                        "description": "",
+                        "name": "rewardAmount",
+                        "type": "STRING"
+                    },
+                    {
+                        "description": "",
+                        "name": "positionFee",
+                        "type": "STRING"
+                    },
+                    {
+                        "description": "",
+                        "name": "keeper",
+                        "type": "STRING"
+                    }
+                ],
+                "name": "accumulationResult",
+                "type": "RECORD"
+            }
+        ],
+        "table_description": "",
+        "table_name": "Market_v2_event_AccountPositionProcessed"
```

Comment:
should these be named the same or different (ie can two jsons map to the same table name?)